ofl/notoemoji/METADATA.pb

```diff
@@ -0,0 +1,68 @@
+name: "Noto Emoji"
+designer: "Google"
+license: "OFL"
+category: "SANS_SERIF"
+date_added: "2022-04-29"
+fonts {
+  name: "Noto Emoji"
+  style: "normal"
+  weight: 400
+  filename: "NotoEmoji[wght].ttf"
+  post_script_name: "NotoEmoji-Regular"
+  full_name: "Noto Emoji Regular"
+  [google.fonts.copyright]: "Copyright  Google Inc. All Rights Reserved."
```

Comment:
Also this entry is weird

Reply:
It looks like the name table defined copyright notice is `Copyright 2013, 2022 Google Inc. All Rights Reserved.`